cdptools/sr_models/webvtt_sr_model.py

```diff
@@ -0,0 +1,173 @@
+                        }
+                    ]
+                }
+                turns.append(turn)
+            # Sentence block is part of recent speaker turn
+            elif turns:
+                turns[-1]['data'].append({
+                    'start_time': sentence['start_time'],
+                    'end_time': sentence['end_time'],
+                    'text': normalized_text
+                })
+
+        return turns
+
+    def transcribe(
+        self,
+        file_uri: Union[str, Path],
+        raw_transcript_save_path: Union[str, Path],
+        timestamped_sentences_save_path: Union[str, Path],
+        timestamped_speaker_turns_save_path: Union[str, Path],
+        **kwargs
+    ) -> SRModelOutputs:
+        # Check paths
+        raw_transcript_save_path = Path(raw_transcript_save_path).resolve()
+        timestamped_sentences_save_path = Path(timestamped_sentences_save_path).resolve()
+        timestamped_speaker_turns_save_path = Path(timestamped_speaker_turns_save_path).resolve()
+
+        # Create timestamped speaker turns transcript
+        timestamped_speaker_turns = self._get_speaker_turns(file_uri)
+
+        # Create raw transcript
+        raw_text = ' '.join([sentence['text'] for turn in timestamped_speaker_turns for sentence in turn['data']])
+        raw_transcript = [{'start_time': timestamped_speaker_turns[0]['data'][0]['start_time'],
+                           'end_time': timestamped_speaker_turns[-1]['data'][-1]['end_time'],
+                           'text': raw_text}]
+
+        # Create timestamped sentences transcript
+        timestamped_sentences = [sentence for turn in timestamped_speaker_turns for sentence in turn['data']]
+
+        # Wrap each transcript in the standard format
+        raw_transcript = self.wrap_and_format_transcript_data(
+            data=raw_transcript,
+            transcript_format=constants.TranscriptFormats.raw,
+            confidence=1
+        )
+
+        timestamped_sentences = self.wrap_and_format_transcript_data(
+            data=timestamped_sentences,
+            transcript_format=constants.TranscriptFormats.timestamped_sentences,
+            confidence=1
+        )
+
+        timestamped_speaker_turns = self.wrap_and_format_transcript_data(
+            data=timestamped_speaker_turns,
+            transcript_format=constants.TranscriptFormats.timestamped_speaker_turns,
+            confidence=1
+        )
+
+        # Write files
+        with open(raw_transcript_save_path, "w") as write_out:
+            json.dump(raw_transcript, write_out)
+
+        with open(timestamped_sentences_save_path, "w") as write_out:
+            json.dump(timestamped_sentences, write_out)
+
+        with open(timestamped_speaker_turns_save_path, "w") as write_out:
+            json.dump(timestamped_speaker_turns, write_out)
+
+        # Return the save path
+        return SRModelOutputs(
+            raw_transcript_save_path,
+            1,
```

Comment:
Can you make these both have the keyword arguments just for my own sanity and future-proofing safety.

Reply:
Yes